### DISCUSSION TITLE
No issue, just a big thank you

### DISCUSSION THREAD

**Discussion by m040601** [Sagnac/streamsave#3] [General]
Thanks for providing this script.
I just found it today and started using it immediately
Been searching for long time for this kind of functionality,
and your script 

- is really usefull
- seems to be very well thought (ex. cases 'continuous', AB, start to point)
- carefully documented (IMPORTANT)
- doesnt impose or force me to options or keybindings, is configurable
- unlike other similar "hacks" it worked flawless at first time, with no surprises, i didn't have to go back and forth to the code or README to understand what was happening

Only got into this error (because when random browsing youtube, I use youtube-dl "worst" quality which sometimes is vp9)

> Errors on some videos if you use the default youtube-dl format selection (e.g. dump-cache won't write vp9 + aac with mp4a tags to mkv)

But even that you had it documented in the README. Bravo !

I still have a lot of testing to do, and hope to give you more feedback from an end user perspective and usability.  Explain my real word use cases and so.

It would be great if you could open up the "Discussions"
https://github.com/Sagnac/streamsave/discussions

a better place to post messages like this.
You might want to close this issue and move it there.

**Comment by Sagnac** [comment]
> Only got into this error (because when random browsing youtube, I use youtube-dl "worst" quality which sometimes is vp9)

Those issues were fixed recently with https://github.com/mpv-player/mpv/commit/643c699f2684987db6073ebe8a6ea76e56c87055 thanks to TheAMM who is in the process of improving mpv's recorder here https://github.com/mpv-player/mpv/pull/8877 which should fix the rest of the issues.

I plan to remove transport stream as a container since the codec_tag fix makes this no longer necessary, leaving the script to choose between MP4, WebM, and MKV. The automatic determination is still useful as some sites only support certain formats (Twitter only accepts h.264 + aac in mp4 AFAIK, and others only accept webm, but not mkv). I'll add an option to force a particular extension though since the compatibility issues have been fixed.

> It would be great if you could open up the "Discussions"

I don't mind if Issues is used to ask questions, provide feedback, make feature requests, or share ideas, etc. but I guess since GitHub added Discussions you're right in that it might be more appropriate to use that as a "forum" instead and use Issues as a bug tracker as it's meant to be.

Thanks for the feedback.
